### DISCUSSION TITLE
Format for the Input.csv we need to provide

### DISCUSSION THREAD

**Discussion by dhineshbabu95** [GoogleChromeLabs/ps-analysis-tool#582] [Support Forum] [answered]
I've provided and URL in row A1 and A2 yet it throws me this error.

The command I used is **npm run cli -- -c C:\Users\dhibabu\Desktop\input_url.csv**" where the CSV is located

<img width="763" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/142570108/d08e583e-262f-44fe-91d8-02ce56e62722">

**Comment by milindmore22** [comment, marked as answer]
Hello @dhineshbabu95 

Thanks for reaching out to us! We noticed from your screenshot that technology analysis seems to be causing problems.

If you have the complete error message, please share it. This will help us diagnose the issue more effectively.

Meanwhile, you can bypass technology analysis by adding the `-nt` flag to your command, Please check the [PSAT CLI wiki](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Command-Line-Interface) for more details.
 
Here's an example: `npm run cli -- -c C:\Users\dhibabu\Desktop\input_url.csv -nt`

By using `-nt`, you're essentially telling the PSAT CLI to skip the technology analysis step.

Please let us know if you have the full error message or if using the `-nt` flag helps!

**Reply by dhineshbabu95** [reply]
Error: TypeError: Invalid URL
    at new Site (C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\node_modules\wappalyzer\driver.js:494:13)
    at Driver.open (C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\node_modules\wappalyzer\driver.js:431:18)
    at C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\dist\cli\procedures\analyzeTechnologiesUrlsInBatches.js:50:62
    at Generator.next (<anonymous>)
    at C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\dist\cli\procedures\analyzeTechnologiesUrlsInBatches.js:23:71
    at new Promise (<anonymous>)
    at __awaiter (C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\dist\cli\procedures\analyzeTechnologiesUrlsInBatches.js:19:12)
    at C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\dist\cli\procedures\analyzeTechnologiesUrlsInBatches.js:49:78
    at Array.map (<anonymous>)
    at C:\Users\dhibabu1\Privacy sandbox tool\ps-analysis-tool\dist\cli\procedures\analyzeTechnologiesUrlsInBatches.js:49:65

Node.js v20.11.1
**is the full error**

And `-nt ` flag helps to fix it
